Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -639,6 +644,14 @@ lemma map_monotone : Monotone (Subgraph.map f) := fun _ _ ↦ map_mono
 theorem map_sup (f : G →g G') (H₁ H₂ : G.Subgraph) : (H₁ ⊔ H₂).map f = H₁.map f ⊔ H₂.map f := by
   ext <;> simp [Set.image_union, map_adj, sup_adj, Relation.Map, or_and_right, exists_or]
 
+@[simp] lemma edgeSet_map (f : G →g G') (H : G.Subgraph) :
+    (H.map f).edgeSet = Sym2.map f '' H.edgeSet := by
```

Comment:
```suggestion
@[simp] lemma edgeSet_map (f : G →g G') (H : G.Subgraph) :
    (H.map f).edgeSet = Sym2.map f '' H.edgeSet := by
```
It looks to me like we'd prefer to have the non-graph version of this:
```
lemma Relation.map_symmetric {r : V → V → Prop} (hr : Symmetric r) (f : V → W) :
    Symmetric (Relation.Map r f f) := by
  rintro _ _ ⟨x, y, hxy, rfl, rfl⟩
  refine ⟨_, _, hr hxy, rfl, rfl⟩

lemma Sym2.map_image_fromRel {r : V → V → Prop} (hr : Symmetric r) {f : V → W} :
    Sym2.map f '' (Sym2.fromRel hr) = Sym2.fromRel (Relation.map_symmetric hr f) := by
```
and deduce edgeSet_map from that (it's a one-line proof).